parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java

```diff
@@ -74,19 +78,28 @@ public static WriterVersion fromString(String name) {
   private final int dictionaryPageSizeThreshold;
   private final WriterVersion writerVersion;
   private final boolean enableDictionary;
+  // TODO - Should this even hold an allocator? this isn't really a property
+  private ByteBufferAllocator allocator;
```

Comment:
I've made it final, still not sure if it belongs here, but it is kind of needed as this spawns objects that now need to take an allocator in their constructors.